backend/cfg/cfg.ml

```diff
@@ -138,7 +142,7 @@ let replace_successor_labels t ~normal ~exn block ~f =
              t.fun_tailrec_entry_point_label and the tailrec entry point
              block has as its predecessors *all* the "tailcall self" blocks. *)
           t.fun_tailrec_entry_point_label <-
```

Comment:
In the case of  Tailcall(Self), t.fun_tailrec_entry_point_label must not be None, so Option.map is fine, but may be worth checking.